server/command.go

```diff
@@ -199,14 +203,56 @@ func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 	if p.canConnect(user) {
 		text += "\n" + strings.ReplaceAll(oAuthHelpText, "|", "`")
 	}
+
+	text += "\n" + strings.ReplaceAll(followStatusHelpText, "$", "`")
+
 	return text, nil
 }
 
+func (p *Plugin) runEnableDisableFollowStatus(user *model.User, args *model.CommandArgs) (string, error) {
+	split := strings.Fields(args.Command)
+
+	if len(split) == 2 { // get and show status
+		enabled, appErr := p.getFollowStatusForUser(user.Id)
+		if appErr == nil {
+			var status string
+			if enabled {
+				status = "on"
+			} else {
+				status = "off"
+			}
+			return fmt.Sprintf("Your current `follow_status` setting is `%v`", status), nil
+		} else {
+			return "Your current `follow_status` setting is not set.", nil
+		}
+	}
+	if len(split) == 3 {
+		var err *model.AppError
+		var text string
+		switch split[2] {
+		case "on":
+			err = p.setFollowStatusForUser(user.Id, true)
+			text = "Automatically following your Zoom status."
+		case "off":
+			err = p.setFollowStatusForUser(user.Id, false)
+			text = "Not following your Zoom status."
+		default:
+			text = fmt.Sprintf("Invalid value `%v`. Accepted values: `on`, `off`.", split[2])
+		}
+		if err != nil {
+			return "Could not set your `follow_status` settings", err
+		}
+		return text, nil
+	} else {
+		return "Incorrect number of arguments. Usage: `/zoom follow_status [on|off]`", nil
+	}
+}
+
 // getAutocompleteData retrieves auto-complete data for the "/zoom" command
 func (p *Plugin) getAutocompleteData() *model.AutocompleteData {
 	available := "start, help"
 	if p.configuration.EnableOAuth && !p.configuration.AccountLevelApp {
-		available = "start, connect, disconnect, help"
+		available = "start, connect, disconnect, follow_status [ on | off ], help"
```

Comment:
0/5 I think we should not show the `follow_status` command's arguments to make this a little more condensed

```suggestion
		available = "start, connect, disconnect, follow_status, help"
```